triage/issue-tracker.rst

```diff
@@ -84,16 +84,44 @@ The submission form has only two fields that you need to fill:
   information as appropriate). In particular, *what version of Python* you
   were using.
 
-Understanding the issue's progress and status
----------------------------------------------
-
-There is a number of additional fields like **Assignees**, **Labels**,
-**Projects**, and **Milestone**. Those are filled by triagers and core
-developers, this is covered in the :ref:`triaging` page. You don't need
+You can tag someone, with :samp:`@{username}` in a comment,
+if you think the issue should be brought to their attention.
+Use the :ref:`experts` to know who wants to be
+tagged or assigned for specific areas.
+
+There are a number of additional fields like **Assignees**, **Labels**,
+and **Projects**. Those are filled by triagers and core
+developers and are covered in the :ref:`triaging` page. You don't need
 to worry about those when reporting issues as a Python user.
 
-You will automatically receive an update each time an action is taken on
-the bug, unless you changed your GitHub notification settings.
+Adding special links
+--------------------
+
+The following abbreviations can be used in a comment to generate a link:
+
+* :samp:`GH-{NNN}`: to link to another issue or PR;
+* :samp:`PEP-{NNN}`: to link to a specific PEP;
+* :samp:`BPO-{NNN}`: to link to a bugs.python.org issue;
+
+See also the `list of autolinks supported by GitHub <autolinks_>`_.
+
+Following issues
+----------------
+
+If you want to subscribe yourself to an issue, click the :guilabel:`🔔 Subscribe`
+button in the sidebar.  Similarly, if you were tagged by somebody else but
+decided this issue is not for you, you might click the :guilabel:`🔕 Unsubscribe`
```

Comment:
```suggestion
decided this issue is not for you, click the :guilabel:`🔕 Unsubscribe`
```

May as well be direct here, if the user has already decided the issue is not for them.